trade-finance-manager-ui/server/controllers/case/underwriting/pricing-and-risk/helpers.js

```diff
@@ -2,6 +2,6 @@ const CONSTANTS = require('../../../../constants');
 const { userIsInTeam } = require('../../../../helpers/user');
 
 const userCanEditGeneral = (user) =>
-  userIsInTeam(user, [CONSTANTS.TEAMS.UNDERWRITERS, CONSTANTS.TEAMS.UNDERWRITER_MANAGERS]);
+  userIsInTeam(user, [CONSTANTS.TEAMS.UNDERWRITERS, CONSTANTS.TEAMS.UNDERWRITER_MANAGERS, CONSTANTS.TEAMS.RISK_MANAGERS]);
```

Comment:
Can we update the test to include the new user? `pricing-and-risk/helpers.test.js `

Reply:
done